### PR TITLE
Added a bit more detail about the JSON console logger

### LINKED ISSUE #27965: Clarify JSON console logger usage
Explain that the JSON console logger doesn't accept JSON strings, but instead will convert messages into JSON on your behalf when logging.

https://docs.microsoft.com/en-us/dotnet/core/extensions/console-log-formatter#json

### PR DESCRIPTION
**Pull request by IEvangelist**
## Summary

Add clarifying details about JSON console logger behavior and expected log messages.

Fixes #27965

✔️ [Preview: Console log formatting—Json](https://review.docs.microsoft.com/en-us/dotnet/core/extensions/console-log-formatter?branch=pr-en-us-27966#json)

### CODE REVIEW

File: docs/core/extensions/console-log-formatter.md [hunk 1]
```diff
@@ -65,21 +65,38 @@ dotnet new webapp -o Console.ExampleFormatters.Json
...
+      Content root path: .\snippets\logging\console-formatter-json
 ```
 
 By default, the `Simple` console log formatter is selected with default configuration. You change this by calling `AddJsonConsole` in the *Program.cs*:
 
 :::code language="csharp" source="snippets/logging/console-formatter-json/Program.cs" highlight="14-22":::
 
+Alternatively, you can also configure this using logging configuration, such as that found in the _appsettings.json_ file:
+
+:::code language="json" source="snippets/logging/console-formatter-json/" highlight="14-24":::
```
Comment (IEvangelist) [inline docs/core/extensions/console-log-formatter.md:88]:
```suggestion
:::code language="json" source="snippets/logging/console-formatter-json/appsettings.json" highlight="14-24":::
```

File: docs/core/extensions/console-log-formatter.md [hunk 2]
```diff
@@ -65,21 +65,38 @@ dotnet new webapp -o Console.ExampleFormatters.Json
...
+      Application started. Press Ctrl+C to shut down.
+info: Microsoft.Hosting.Lifetime[0]
+      Hosting environment: Development
+info: Microsoft.Hosting.Lifetime[0]
+      Content root path: .\snippets\logging\console-formatter-json
 ```
 
 By default, the `Simple` console log formatter is selected with default configuration. You change this by calling `AddJsonConsole` in the *Program.cs*:
 
 :::code language="csharp" source="snippets/logging/console-formatter-json/Program.cs" highlight="14-22":::
```
Comment (IEvangelist) [inline docs/core/extensions/console-log-formatter.md:84]:
```suggestion
:::code language="csharp" source="snippets/logging/console-formatter-json/Program.cs" highlight="7-15":::
```

File: docs/core/extensions/console-log-formatter.md [hunk 3]
```diff
@@ -65,21 +65,38 @@ dotnet new webapp -o Console.ExampleFormatters.Json
...
 
+Alternatively, you can also configure this using logging configuration, such as that found in the _appsettings.json_ file:
+
+:::code language="json" source="snippets/logging/console-formatter-json/appsettings.json" highlight="14-24":::
```
Comment (IEvangelist) [inline docs/core/extensions/console-log-formatter.md:88]:
```suggestion
:::code language="json" source="snippets/logging/console-formatter-json/appsettings.json" highlight="14-23":::
```

**Review by BillWagner — APPROVED** [review]
This LGTM @IEvangelist 

You can :shipit: now.

**Pull request merged**